src/vivarium_gates_mncnh/data/loader.py

```diff
@@ -659,6 +664,61 @@ def load_no_antibiotics_paf(
     return utilities.set_non_neonnatal_values(paf_no_antibiotic, 0.0)
 
 
+def load_probiotics_facility_probability(
+    key: str, location: str, years: Optional[Union[int, str, List[int]]] = None
+) -> float:
+    # Currently no coverage
+    return 0.0
+
+
+def load_no_probiotics_relative_risk(
+    key: str, location: str, years: Optional[Union[int, str, List[int]]] = None
+) -> pd.DataFrame:
+    # Relative risk for no receieving intervention
+    rr_dist = data_values.PROBIOTICS_RELATIVE_RISK_DISTRIBUTION
+    demography = get_data(data_keys.POPULATION.DEMOGRAPHY, location)
+    draws = get_random_variable_draws(metadata.ARTIFACT_COLUMNS, key, rr_dist)
+    data = pd.DataFrame([draws], columns=metadata.ARTIFACT_COLUMNS, index=demography.index)
+    data.index = data.index.droplevel("location")
+
+    return utilities.set_non_neonnatal_values(data, 1.0)
```

Comment:
what does the 1.0 mean here?

Reply:
A rr of 1.0 is no risk. It is the rr equivalent of a 0.0 exposure value.